homeassistant/components/atag/__init__.py

```diff
@@ -0,0 +1,164 @@
+"""The ATAG Integration."""
+from datetime import timedelta
+
+from pyatag import SENSOR_TYPES, AtagDataStore, AtagException
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    DEVICE_CLASS_PRESSURE,
+    DEVICE_CLASS_TEMPERATURE,
+    EVENT_HOMEASSISTANT_STOP,
+)
+from homeassistant.core import HomeAssistant, asyncio, callback
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers import dispatcher
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.event import async_track_time_interval
+
+DOMAIN = "atag"
+DATA_LISTENER = "atag_listener"
+SIGNAL_UPDATE_ATAG = "atag_update"
+PLATFORMS = ["sensor", "climate", "water_heater"]
+DEFAULT_SCAN_INTERVAL = timedelta(seconds=30)
+UNIT_TO_CLASS = {"°C": DEVICE_CLASS_TEMPERATURE, "Bar": DEVICE_CLASS_PRESSURE}
```

Comment:
Do you mean from local const.py or homeassistant.const.py? i.e. should I import `CONF_PLATFORM` from homeassistant and then reassign? or create a local const.py? Because these strings aren't used in other files and the devdocs say `Only add new constants to const.py if they are widely used. Otherwise keep them on components level
`.
Therefore I tried to limit the creation of an extra file for imports if not needed in multiple files

Reply:
imported the units and bar - hope that's what you referred to?

Reply:
Yeah exactly, sorry should have been a bit more precise 😅